recipes/rnaformer/meta.yml

```
@@ -0,0 +1,57 @@
+{% set name = "RNAformer" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/57/19/1fc85e4b301f39994f6f7e14657e856a6e2efd200f484eeed0e0e5e2e2b0/rnaformer-0.0.1.tar.gz
```

```suggestion
   url: https://pypi.io/packages/source/r/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
```
this will make it possible the BiocondaBot to update the recipe when a new version is released